cts.schema.json

```
@@ -112,7 +122,19 @@
       "type": "boolean",
       "const": true,
       "description": "The flag indicating that the selector is not a valid JSONPath selector expression"
+    },
+    "result_paths": {
+      "description": "The expected normalized JSONPath query strings, one for each result value",
```

I think this could be made clearer:
```suggestion
      "description": "The expected normalized JSONPath query strings of applying the selector to the document, contains all the normalized JSONPath query strings of matched values",
```